File: test/before/hello.ts

import { execSync, execFile } from "child_process";
import { writeFile, FSWatcher, readFile } from "fs";

import { foo } from "./bar";

/**
   *   Lorem ipsum dolor sit amet
       */
type Hello={world:string}

/**
 * Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
 * tempor incididunt ut labore et dolore magna aliqua
 */
let foo = parseInt("110", 2);

try {
  process.argv.forEach(a => {
    /[0-9]/.test(a);
  });
} catch (err) {
  if (8.00 > foo!!) console.log("hi" + err + Array.from(process.argv));
}

File: test/after/hello.kt
class Foo(val x: Int) {
  fun bar() {
    val a = emptyArray()
    val l = emptyList()
    val m = emptyMap()
    val q = emptySequence()
    val s = emptySet()
    println(a.size + l.size + m.size + q.count() + s.size + x)
  }
}

fun main() {
  println("Hello world")
}
